### PR TITLE
docs: ✏️ add docs/script for running single browser testcafe

### PR DESCRIPTION
**Pull request by maniator**
---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,16 @@ npm run test
 To run (TestCafe) browser integration tests:
 
 ```
-npm run integ:local:chrome:headless
+npm run integ:headless
+npm run integ:browser
+```
+
+To run (TestCafe) integration tests for a specific broswer:
```
Comment (adebayor123) [inline README.md:106]:
```suggestion
To run (TestCafe) integration tests for a specific browser:
```

File: README.md [hunk 2]
```diff
@@ -99,7 +99,16 @@ npm run test
...
+
+```
+npm run integ -- (browserName)
+# for example
+npm run integ -- chrome:headless
```
Comment (adebayor123) [inline README.md:111]:
```suggestion
npm run integ -- chrome:headless
npm run integ -- chrome
```

**Review by adebayor123 — COMMENTED** [review]
I think we can add those two changes

**Comment by maniator** [comment]
> I think we can add those two changes

Done @adebayor123

File: package.json [hunk 3]
```diff
@@ -37,11 +37,11 @@
         "prettier": "npx prettier --check .",
         "prettier:fix": "npm run prettier -- --write",
         "prepublishOnly": "npm run release",
-        "preinteg:browser": "npm run build:dev",
-        "integ:browser": "testcafe --config-file=./.testcafe.browser.js --app \"http-server ./build/dev -s\"",
-        "integ:browser:ie": "testcafe ie --app \"http-server ./build/dev -s\" src/__integ__/unsupportedBrowser.test.ts",
-        "preinteg:headless": "npm run build:dev",
-        "integ:headless": "testcafe --config-file=./.testcafe.headless.js --app \"http-server ./build/dev -s\"",
+        "preinteg": "npm run build:dev",
```
Comment (adebayor123) [inline package.json:40]: Doesn't your previous PR address these changes already?

Comment (maniator) [reply package.json:40]: Nope.

This gets rid of the duplicate `pre<x>` steps and also allows for you to pass a browser (in main if you try, it first read the config file, which then errors out)

so now both headless and browser integ scripts inherit from one point

**Review by adebayor123 — APPROVED** [review]

**Review by qhanam — APPROVED** [review]

**Pull request merged**